packages/server/test/parse-gitignore.test.ts

```diff
@@ -0,0 +1,25 @@
+import parseGitignore from 'parse-gitignore'
```

Comment:
Why do we need to test an external library?

Reply:
- to validate and fixate our assumptions about its behaviour. Gives you peace of mind when upgrading its version down the road
- to have an easy-to-reproduce playground for testing out libraries

Reply:
Hmm this is kind of already tested here: https://github.com/jonschlinkert/parse-gitignore/blob/master/test/test.js Perhaps it makes more sense to unit test this library in the context of the way we are using it?

Reply:
We can do that. There's a test in https://github.com/blitz-js/blitz/pull/365/commits/2648d55b53be210b8e6656ef3c1430894263cc29, I'll remove this one.

Also, I googled what these kinds of tests are called again: Contract tests.